src/utils/tickets.js

```diff
@@ -81,7 +83,13 @@ const processTicket = async ticket => {
   artifacts.push(...ssArtifacts);
 
   // process resources
-  artifacts.push(...(await resourceManager.process()));
+  const jsArtifacts = await resourceManager.process();
+  artifacts.push(...jsArtifacts);
+
+  // scan js
+  yaraManager.setupResourceScan(jsArtifacts, ticket);
+  const yaraArtifacts = await yaraManager.process();
+  artifacts.push(...yaraArtifacts);
```

Comment:
Nice following of the paradigm! Everything here looking the same makes it real easy to read.